src/ecdsa/ecdh.py

```diff
@@ -97,39 +108,123 @@ def load_private_key(self, private_key):
         return self.private_key.get_verifying_key()
 
     def load_private_key_bytes(self, private_key):
+        """
+        Load private key from plain bytes string.
+        Uses current curve and checks if key length corresponds to
+        the current curve.
+        :param private_key: private key in bytes string format
+        :return: public (verifying) key from this private key
+        """
         return self.load_private_key(
             SigningKey.from_string(private_key, curve=self.curve))
 
     def load_private_key_der(self, private_key):
+        """
+        Load private key from DER string.
+        Curve is get from DER and check if it the same as current curve
+
+        Note, the only DER format supported is the RFC5915
+        Look at keys.py:SigningKey.from_der()
+
+        :param private_key: binary string with the DER encoding of private ECDSA key
+        :return: public (verifying) key from this private key
+        """
         return self.load_private_key(SigningKey.from_der(private_key))
 
     def load_private_key_pem(self, private_key):
+        """
+        Load private key from PEM string.
+        Curve is get from PEM and check if it the same as current curve
+
+        Note, the only PEM format supported is the RFC5915
+        Look at keys.py:SigningKey.from_pem()
+        it needs to have `EC PRIVATE KEY` section
+
+        :param private_key: text with PEM-encoded private ECDSA key
+        :return: public (verifying) key from this private key
+        """
         return self.load_private_key(SigningKey.from_pem(private_key))
 
     def get_public_key(self):
+        """
+        Get public key from local private key.
+        Usually needs to send it to remote party.
```

Comment:
"usually"?

also, I think it should be "needs to be sent to the remote party"?